ffconv/file_processor.py

```diff
@@ -57,28 +33,36 @@ def _build_merge_params(streams, inputs, maps, meta):
     @staticmethod
     def _build_merge_command(inputs, maps, meta, output):
         """
-        Build merge command (if we need to make a conversion) as a list of strings.
+        Build merge command (if we need to make a conversion) as a list of
+        strings.
         """
         cmd = []
         if len(inputs) > 1:
             # Build merge command and execute it
             cmd.append('ffmpeg')
+
+            # Add each of the resulting inputs
             for input in inputs:
                 cmd.extend(['-i', input])
+
+            # Add each of the maps
             for map in maps:
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Redefining built-in 'map' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=Pylint%3AW0622)